bedrock/firefox/templates/firefox/new/horizon/base.html

```diff
@@ -2,7 +2,7 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% add_lang_files "firefox/new" %}
+{% add_lang_files "firefox/new/horizon" %}
```

Comment:
@peiying2 - can you please let me know if this is an OK location for the new .lang file?